ucp/_libs/send_recv.pyx

```diff
@@ -39,7 +39,7 @@ cdef void _send_callback(void *request, ucs_status_t status):
         req.finished = True
         return
     cdef object future = <object> req.future
-    if future.get_loop().is_closed():
+    if asyncio.get_event_loop().is_closed():
```

Comment:
@madsbk is this safe?  

In particular, will we only call this function from within the thread running the event loop?

Reply:
Yes, it should be fine

Reply:
Thanks @madsbk